SecurityManager/Clients.Web.Test/App.config

```diff
@@ -54,8 +60,11 @@
       </customErrors>
       -->
     <customErrors mode="Off" />
-    <httpRuntime requestValidationMode="2.0" targetFramework="4.8" />
+    <httpRuntime requestValidationMode="2.0"/>
     <globalization requestEncoding="utf-8" responseEncoding="utf-8" culture="de-AT" uiCulture="de-AT" />
+    <httpHandlers>
+      <add verb="*" path="*.wxe" type="Remotion.Web.ExecutionEngine.WxeHandler, Remotion.Web" />
+    </httpHandlers>
   </system.web>
   <system.webServer>
```

Comment:
I don't think we need the `system.webServer` tag in the `App.config`. This was read by IIS in the `Web.config` and used for running in integrated mode but should no longer be necessary because we don't have the same IIS integration. This is also the reason why you had to move the `handlers` section up. 

We also might need the other registered handler for something.